src/commands/flow/execute.ts

```diff
@@ -413,16 +417,14 @@ export async function execute(
               )
             }
 
-            let csvData: any = await readFile(csvFileRealPath).catch((err) =>
+            let csvContent = (await readFile(csvFileRealPath).catch((err) =>
               displayError(err, 'Error while reading CSV file.')
```

Comment:
`csvContent` will be undefined when there's an error caught from `readFile`.
It will make the `csvContent.split` on line 424 fail.

Reply:
You might want to return `''` from inside the catch block to make it safe.

Reply:
good suggestion!

Reply:
done